src/util/invariant.h

```diff
@@ -51,14 +53,63 @@ Author: Martin Brain, martin.brain@diffblue.com
 ** CPROVER_INVARIANT_* macros.
 */
 
+/// A logic error, augmented with a distinguished field to hold a backtrace.
 class invariant_failedt: public std::logic_error
 {
+  /// Stores the backtrace set by `invariant_violated`.
```

Comment:
Normally I would explicitly write `private:` rather than relying on the default in case someone adds something public above.